xcdat/spatial.py

```diff
@@ -716,6 +725,9 @@ def _averager(self, data_var: xr.DataArray, axis: List[SpatialAxis]):
             Data variable inside a Dataset.
         axis : List[SpatialAxis]
             List of axis dimensions to average over.
+        required_weight : optional, float
```

Comment:
Is `required_weight` a good parameter name? What should the default be (currently zero to make default behavior backwards compatible).

Reply:
Maybe `required_weight_pct`?  `required_weight` sounds like a boolean parameter.

I think 0 is the appropriate default value.

Reply:
Actually I like `None` as a default value to indicate the arg is not set by default.

Reply:
`min_weight` is good with me!